travis/deploy.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+chmod 600 ./travis/id_rsa
+
+read -r -d '' DEPLOY_SCRIPT << EOM
+  sudo su;
+  cd /home/ubuntu/brokernode;
+  git stash;
+  git clean -f -d;
+  git checkout master;
+  git pull;
+  cp /home/ubuntu/database.dev.yml ./database.yml;
+  cp /home/ubuntu/docker-compose.dev.yml ./docker-compose.yml;
+  docker stop $(docker ps -aq);
+  docker rm $(docker ps -aq);
+  docker rmi $(docker images -q);
+  sudo curl https://gist.githubusercontent.com/mlebkowski/471d2731176fb11e81aa/raw/4a6a18a19ac5a9f1b8c6533a07f93e01da8ddba0/cleanup-docker.sh | bash;
```

Comment:
I'm going to go ahead and approve this, but we should consider whether we want to do 
`cp .env.test .env` before we do:
`sudo sed -i -e 's/127.0.0.1/$real_ip/' ./.env`

There's not, afaik, an obvious way to tell the dev brokers when we have added a new variable option to the .env vars, so the only good way I could think of to do it would be to check in some default value to .env.test and that'd get copied over.  

Also not sure, but this:
`sudo sed -i -e 's/127.0.0.1/$real_ip/' ./.env`
might need to be this:
`sudo sed -i -e 's/127.0.0.1/$real_ip/' .env`